src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -235,40 +235,29 @@ private void setupKeyboardShortcuts() {
                 KeyPress.setLastKeyPressedCodeAndTime(event.getCode());
             }
             if (NEW_COMMENT.match(event)) {
-                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else {
-                    if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
-                        openPageOfCurrentlySelectedIssue();
-                    } else if (!ui.getBrowserComponent().isCurrentUrlDiscussion()) {
-                        ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
+                    return;
```

Comment:
Why do we return early here? Based on the old code, this `return` will change the behaviour completely, because in the `else` part we will call `jumpToComment` for any case.